nautobot/ui/package.json

```diff
@@ -60,5 +60,15 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "^@components(.*)$": "<rootDir>/src/components$1",
```

Comment:
Any reason this first line anchors to the beginning and uses `<rootDir>` but none of the others do?